qemu/tests/cfg/device_assignment.cfg

```diff
@@ -7,7 +7,20 @@
     variants:
         - multi_pf_boot:
             type = boot
-            nics = "nic1 nic2"
+            nics = "nic1 nic2 nic3"
```

Comment:
does the script works? seems we have give up to develop the script about device assignment